packages/kbn-apm-synthtrace/src/cli/run_synthtrace.ts

```diff
@@ -8,17 +8,25 @@
 import datemath from '@kbn/datemath';
 import { Argv } from 'yargs';
 import yargs from 'yargs/yargs';
+import { readdirSync } from 'fs';
+import path from 'path';
 import { intervalToMs } from './utils/interval_to_ms';
 import { parseRunCliFlags } from './utils/parse_run_cli_flags';
 import { startHistoricalDataUpload } from './utils/start_historical_data_upload';
 import { startLiveDataUpload } from './utils/start_live_data_upload';
 
+function getBuiltinScenarios() {
+  return readdirSync(path.resolve(__dirname, '../scenarios')).map((s) => s.replace(/\.ts$/, ''));
+}
+
 function options(y: Argv) {
   return y
+    .usage('$0 <file>')
     .positional('file', {
-      describe: 'File that contains the trace scenario',
+      describe: 'Path to file that contains the scenario or name of a built-in scenario',
```

Comment:
Good catch on remove the trace from the message.
Regarding adding option to specify a different path, i am not sure.

Idea is to have all the scenarios present in the same folder so that anyone can reuse them for their use case, like APM or Logs or Infra team.

I would keep this part simple. `Scenario file name` or additionally we can show the folder path as to where the scenario files are for the user to find them

Reply:
My understanding is that `synthtrace` accepts an arbitrary path on your file system, or alternatively one of the built-in scenarios. I didn't want to change the functionality of the tool so kept that part as is since it might be useful when loading scenario files from outside the Kibana codebase.

Reply:
Yes, else we will lose the reusability and most importantly in the last PR i introduced a breaking change for which i had to update all the Scenario files. As they all were co-located it was super easy to update them. I would restrict them to be in the same folder.

Reply:
How would you go about making this breaking change? 

Are there any external consumers of this tool? (i.e. is it only used by developers for local development purposes or are there any CI builds / tests that use the tool as well?)

Reply:
> How would you go about making this breaking change?

I didn't understand this ?

> Are there any external consumers of this tool?

The CLI runner of the tool is used only by developers from different teams

Reply:
TBH, it does not matter if a developer wants to run a scenario file present anywhere. By enforcing them to the same folder, 
- We prevent them from writing scenario files, which could already be present, saving effort
- They might add a new scenario which could be easily shared between other devs.
- Tomorrow we introduce something which could impact how scenario files are written, it would be easy to update the scenario files

Reply:
> > How would you go about making this breaking change?
> 
> I didn't understand this ?

Removing the ability to load a scenario file from any location (which `synthtrace` currently supports) is a breaking change. A developers or an existing system might rely on this feature so there would have to be some kind of deprecation process. 

I was wondering how you handle such changes usually.

Reply:
Currently we don't support loading Scenario files outside Kibana Workspace.
And i highly doubt we have scenario files in any other folder inside Kibana apart from here.
It's safe to make this explicit.

Reply:
Awesome, thanks, I've updated the parameter according!